docs/running-a-node/beta-network.md

```diff
@@ -92,19 +83,44 @@ docker run --restart=unless-stopped -d \
 
 | URL                                     | Description |
 |                                         |             |
-| https://beta.nano.org/                  | Official beta site and faucet |
+| https://beta.nano.org/                  | Nano Foundation beta site and faucet |
 | https://beta.nanocrawler.cc/            | Beta Explorer |
-| https://b.repnode.org/                  | Beta nodes and Stats |
+| https://beta.nanoticker.info/           | Beta node details and stats |
+| https://b.repnode.org/                  | Beta node details and stats |
+
+## Differences from the main network
 
+| Parameter | Main Network | Beta Network | Comment |
+|-----------|--------------|--------------|---------|
+| [Proof of Work](/integration-guides/the-basics/#proof-of-work) Difficulty Threshold | `0xffffffc000000000` | `0xfffff00000000000` | 64 times lower on the beta network |
```

Comment:
This information doesn't cover different epochs, should we have 3 different entries in this table?

- Epoch 1 `0xfffff00000000000` (64x lower than main epoch 1)
- Epoch 2 `0xfffff80000000000` (2x higher than beta epoch 1)
- Epoch 2 receive `0xffffe00000000000` (2x lower than beta epoch 1)

Reply:
```suggestion
| Epoch 1 difficulty threshold | `0xffffffc000000000` | `0xfffff00000000000` | 64 times lower on the beta network |
| Epoch 2 send/change threshold | `0xfffffff800000000` | `0xfffff00000000000` | 2 times higher than epoch 1 |
| Epoch 2 receive threshold | `0xfffffe0000000000` | `0xffffe00000000000` | 2 times lower than epoch 1 |
```